test/icinga-perfdata.cpp

```diff
@@ -43,6 +43,15 @@ BOOST_AUTO_TEST_CASE(multiple)
 	BOOST_CHECK(str == "testA=123456 testB=123456");
 }
 
+BOOST_AUTO_TEST_CASE(multiline)
```

Comment:
Test at least has to also be added to https://github.com/Icinga/icinga2/blob/e31bb2b534bf45cb830d54708a78ecd1fc88d257/test/CMakeLists.txt#L46. Apart from that, it's unclear why this is named "multiline" without a single `\n` in the test.